app/bundles/ReportBundle/Helper/ReportHelper.php

```diff
@@ -2,8 +2,19 @@
 
 namespace Mautic\ReportBundle\Helper;
 
+use Mautic\ReportBundle\Event\ColumnCollectEvent;
+use Mautic\ReportBundle\ReportEvents;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
 final class ReportHelper
 {
+    private EventDispatcherInterface $dispatcher;
+
+    public function __construct(EventDispatcherInterface $dispatcher)
+    {
+        $this->dispatcher = $dispatcher;
+    }
```

Comment:
Since we ditched PHP 7.4 we can use https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion 🎉 
```suggestion
    public function __construct(private EventDispatcherInterface $dispatcher)
    {
    }
```